pkg/management/postgres/webserver/webserver.go

```diff
@@ -118,12 +118,23 @@ func (ws *Webserver) Start(ctx context.Context) error {
 	return nil
 }
 
+func sendTextResponse(w http.ResponseWriter, statusCode int, message string) {
+	w.Header().Set("Content-Type", "text/plain")
+	w.WriteHeader(statusCode)
```

Comment:
I'm interested if explicitly sending the headers will help us here (as my debugging indicates we get nothing after the TLS handshake).

It's curious that we get `succeeding` logs for `/pg/status` despite the hang.